### PR TITLE
feat(ingest): GE Profile/Action Trino support

### PR DESCRIPTION
**Pull request by aezomz**
Should resolve this too
https://feature-requests.datahubproject.io/p/profiling-in-trino

Bumping GE version due to major fix on Trino connector [here](https://github.com/great-expectations/great_expectations/pull/5261).

Feature update:
Trino will delete GE create temporary views
GE Actions with two new parameter refer to Documentation:
    - `exclude_dbname` 
    - `platform_alias`

Test update to include those two new parameters

`platform_alias` will be useful, when our main connector is `presto-on-hive` and we are capitalizing on `trino` to run test and profiling

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by aezomz** [comment]
Most of the error are from this, 
```
E       from __future__ import annotations
E                                        ^
E   SyntaxError: future feature annotations is not defined
```
https://stackoverflow.com/a/61544901
which need at least python 3.7. 
I guess using GE will need at least 3.7 python version.

Having some CICD Integration errors too.

File: metadata-ingestion/setup.py [hunk 1]
```diff
@@ -90,9 +90,9 @@ def get_long_description():
     # Required for all SQL sources.
     "sqlalchemy==1.3.24",
     # Required for SQL profiling.
-    "great-expectations>=0.14.11,<0.15.3",
-    # datahub does not depend on Jinja2 directly but great expectations does. With Jinja2 3.1.0 GE 0.14.11 is breaking
-    "Jinja2<3.1.0",
+    "great-expectations>=0.15.12",
```
Comment (hsheth2) [inline metadata-ingestion/setup.py:93]: great-expectations actually dropped support for Python 3.6, and so requiring `>=0.15.12` isn't compatible with DataHub at the moment.

However, we're also working on dropping support for Python 3.6 (see https://github.com/datahub-project/datahub/pull/5521), so this won't be an issue for too much longer.

We've got two options then:
1. Wait until datahub drops 3.6 and then merge this PR without issue
2. Modify this PR to use a GE release that is compatible with 3.6

I think the former option makes more sense, but I know that this PR has been sitting for a while and it's annoying to wait even more.

Comment (aezomz) [reply metadata-ingestion/setup.py:93]: Hi! Thanks for taking up this PR. I agree that the former is a better choice here. Lets come back to this when datahub drops 3.6 .

This PR is highly dependent on the fix i made in this release for GE [here](https://github.com/great-expectations/great_expectations/releases/tag/0.15.11) which is at least 0.15.11 .

**Comment by hsheth2** [comment]
Hey @aezomz, we just merged in #5521 so we should be clear to finish up this PR

**Comment by aezomz** [comment]
Thanks, let me fix those failure tonight.

**Review by hsheth2 — APPROVED** [review]
LGTM

I'm not a huge fan of the `exclude_dbname` / alias config being in the profiler as well. Ideally we should just pass the URNs from the sql source into the profiler, but that's a bigger, unrelated task cc @shirshanka

**Comment by aezomz** [comment]
Are we intending to merge this soon?

**Comment by hsheth2** [comment]
@aezomz just merged, sorry about the delay.

**Pull request merged**